plugins/ltac/tacenv.ml

```diff
@@ -158,6 +158,7 @@ let load_md i ((sp, kn), (local, id, b, t, deprecation)) = match id with
 
 let open_md i ((sp, kn), (local, id, b, t, deprecation)) = match id with
 | None ->
+  (* todo: generate a warning when non-unique, record choices for non-unique mappings *)
```

Comment:
This todo is irrelevant, since there is no attached code to it. Furthermore it's probably not even the right place, so please remove it.